Unix/samples/Providers/Indication/Lifecycle/XYZ_ComputerSystem.c

```diff
@@ -10,6 +10,7 @@
 /* @migen@ */
 #include <MI.h>
 #include <util.h>
+#include <pal/sem.h>
```

Comment:
I don't fully understand, sorry. You're including this include file, but I don't see it referenced in code anywhere (i.e. actually calling it). Am I missing something?